SimpleChat.Client/Shared/UserComponent.razor.cs

```diff
@@ -12,6 +12,7 @@ public partial class UserComponent
     {
         private const string UserNameKeyName = "UserName";
         private const string UserImgKeyName = "UserImgUrl";
+        private const string UserIdKeyName = "UserId";
```

Comment:
If you use same string constants in different places - put them in one separate class